### PR TITLE
Iia Hnatiuk: Cypress registration form

### PR DESCRIPTION
**Pull request by iiahnatiuk**

### CODE REVIEW

File: cypress/support/commands.js [hunk 1]
```diff
@@ -23,3 +23,11 @@
...
 // Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
+
+Cypress.Commands.add('findByPlaceholder', (placeholder) => {
+  cy.get(`[placeholder="${placeholder}"]`);
+});
+
+Cypress.Commands.add('pickDate', (date) => {
+  cy.get(`.react-datepicker__${date}`);
+});
\ No newline at end of file
```
Comment (dashaprotsko) [inline cypress/support/commands.js:33]: Add a newline at the end of the file

File: cypress.config.js [hunk 2]
```diff
@@ -1,8 +1,33 @@
...
 
 module.exports = defineConfig({
   e2e: {
+    viewportHeight: 1080,
+    viewportWidth: 1980,
     setupNodeEvents(on, config) {
+      on("task",{
+        generateUser() {
+          genders = ['Female', 'Male', 'Other'];
+          subjects = ['English', 'Chemistry', 'Computer Science'];
```
Comment (dashaprotsko) [inline cypress.config.js:12]: Declare the arrays 'genders', 'subjects', and 'hobbies' as constants outside of the 'generateUser' function to avoid redeclaring them each time the function runs.

File: cypress/e2e/registration.cy.js [hunk 3]
```diff
@@ -1,11 +1,71 @@
...
+    cy.task('generateUser')
+      .then(generateUser => {
+        user = generateUser;
+      })
   });
 
-  it('', () => {
-    
+  it('should register a new student', () => {
+    cy.findByPlaceholder('First Name')
```
Comment (dashaprotsko) [inline cypress/e2e/registration.cy.js:15]: Consider adding tests for edge cases and negative scenarios, such as missing required fields or invalid input.

**Review by burovetskyi — APPROVED** [review]
Good Job!

**Review by dashaprotsko — CHANGES_REQUESTED** [review]
Some fixes are required
